packages/siri-sx-generator/test/siriSXUploadToDynamo.ts

```diff
@@ -1,9 +1,10 @@
-import { PutCommand } from "@aws-sdk/lib-dynamodb";
+/* eslint-disable no-console */
```

Comment:
why do we have this for the whole file?

Reply:
There are 2 console.logs - ill remove the inline ones.